conans/assets/templates/search_table_html.py

```diff
@@ -0,0 +1,96 @@
+
+content = """
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Conan | {{ search.reference }}</title>
+        <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/dt/dt-1.10.20/datatables.min.css"/>
```

Comment:
This was a problem for Conan running without external internet access. We did a trick for the js and css of the graph output, but probably deserves a proper solution, embedding the resources.

Reply:
The graph output is looking for those files in the cache and, if they are not there, it will use Internet ones. If the users are afraid of not having internet, they can use a different template from their `<cache>/templates/` folder and that template can use other URLs or local files.

Nevertheless, this output without CSS and JS is still a HTML table, you don't have filters and so on, but still a table. The problem with the graph is different, without those js/css you see nothing.

Reply:
Yes, but it would be breaking behavior. Users currently rely on the documented way to provide the visjs files for the ``conan info --graph=file.html``  command: https://docs.conan.io/en/latest/reference/commands/consumer/info.html

Reply:
Yes, the proposal for `graph` output is fixed. For this one, I think it is ok to show the plain HTML table and document (if we want to) how to change the template.

Reply:
Yes, sure, as long as it works and it is only a matter of style, good for me.

Reply:
![_Users_jgsogo_dev_conan_tmp_datatables_output html](https://user-images.githubusercontent.com/1406456/79050804-be387e00-7c2c-11ea-992a-c5e3af5b7af4.png)